includes/Handlers/Connection.php

```diff
@@ -381,13 +381,19 @@ public function handle_disconnect() {
 			wp_die( esc_html__( 'You do not have permission to uninstall Facebook Business Extension.', 'facebook-for-woocommerce' ) );
 		}
 		try {
-			$response = facebook_for_woocommerce()->get_api()->get_user();
-			$id       = $response->get_id();
-			if ( null !== $id ) {
-				$response = facebook_for_woocommerce()->get_api()->delete_user_permission( (string) $id , 'manage_business_extension' );
+			$external_business_id = $this->get_external_business_id();
+			if ( null !== $external_business_id ) {
```

Comment:
Should we use a loose "!=" comparison to cover case like empty string? 
```suggestion
			if ( null != $external_business_id ) {
```

Reply:
amended changes in my next commit